### PR TITLE
Adds new descriptions to several cartridges

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Includes new descriptions for several cartridges making it clear wich systems they include.
## Why It's Good For The Game

Most cartridges had no custom descriptions and trying to judge by the name alone what they do can be a difficult task.

## Testing

- Verified new descriptions in game.
## Changelog
:cl:
tweak: Added new descriptions to several cartridges.
/:cl:

### CODE REVIEW

File: code/modules/pda/cart.dm [hunk 1]
```diff
@@ -153,14 +162,16 @@
 	radio = new /obj/item/integrated_radio/mule(src)
 
 /obj/item/cartridge/head
-	name = "Easy-Record DELUXE"
+	name = "Easy-Record"
+	desc = "A data cartridge for portable microcomputers. Has status display controller."
```
Comment (Sirryan2002) [inline code/modules/pda/cart.dm:166]:
```suggestion
	desc = "A data cartridge for portable microcomputers. Has a status display controller."
```

File: code/modules/pda/cart.dm [hunk 2]
```diff
@@ -153,14 +162,16 @@
...
 	icon_state = "cart-h"
 	programs = list(
 		new /datum/data/pda/app/status_display
 	)
 
 /obj/item/cartridge/hop
 	name = "HumanResources9001"
+	desc = "A data cartridge for portable microcomputers. Has supply records, MULEbot control, custodial locator and status display controller."
```
Comment (Sirryan2002) [inline code/modules/pda/cart.dm:174]:
```suggestion
	desc = "A data cartridge for portable microcomputers. Has supply records, MULEbot control, a custodial locator and a status display controller."
```

File: code/modules/pda/cart.dm [hunk 3]
```diff
@@ -176,6 +187,7 @@
 
 /obj/item/cartridge/hos
 	name = "R.O.B.U.S.T. DELUXE"
+	desc = "A data cartridge for portable microcomputers. Has security records, security bot control and status display controller."
```
Comment (Sirryan2002) [inline code/modules/pda/cart.dm:190]:
```suggestion
	desc = "A data cartridge for portable microcomputers. Has security records, security bot control and a status display controller."
```

File: code/modules/pda/cart.dm [hunk 4]
```diff
@@ -199,6 +212,7 @@
 
 /obj/item/cartridge/cmo
 	name = "Med-U DELUXE"
+	desc = "A data cartridge for portable microcomputers. Has medical records, reagent scanner, med scanner and status display controller."
```
Comment (Sirryan2002) [inline code/modules/pda/cart.dm:215]:
```suggestion
	desc = "A data cartridge for portable microcomputers. Has medical records, a reagent scanner, med scanner and status display controller."
```

File: code/modules/pda/cart.dm [hunk 5]
```diff
@@ -209,6 +223,7 @@
 
 /obj/item/cartridge/rd
 	name = "Signal Ace DELUXE"
+	desc = "A data cartridge for portable microcomputers. Has a reagent scanner, gas scanner status display controller and signaler system."
```
Comment (Sirryan2002) [inline code/modules/pda/cart.dm:226]:
```suggestion
	desc = "A data cartridge for portable microcomputers. Has a reagent scanner, gas scanner, a status display controller and signaler system."
```

File: code/modules/pda/cart.dm [hunk 6]
```diff
@@ -247,6 +262,7 @@
 
 /obj/item/cartridge/supervisor
 	name = "Easy-Record DELUXE"
+	desc = "A data cartridge for portable microcomputers. Has security records and status display controller."
```
Comment (Sirryan2002) [inline code/modules/pda/cart.dm:265]:
```suggestion
	desc = "A data cartridge for portable microcomputers. Has security records and a status display controller."
```

File: code/modules/pda/cart.dm [hunk 7]
```diff
@@ -277,6 +293,7 @@
 
 /obj/item/cartridge/syndicate
 	name = "Detomatix Cartridge"
+	desc = "Allows you to use the detonate function in the messenger program to attempt to remotely detonate other people's PDAs."
```
Comment (Sirryan2002) [inline code/modules/pda/cart.dm:296]:
```suggestion
	desc = "Allows you to remotely detonate other people's PDAs through the messenger program."
```

**Comment by Sirryan2002** [comment]
In the future you can batch suggestions into a single commit

**Review by Sirryan2002 — APPROVED** [review]

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**